IBAnimatable/BorderDesignable.swift

```diff
@@ -25,39 +25,70 @@ public protocol BorderDesignable {
 
 public extension BorderDesignable where Self: UIView {
   public func configBorder() {
-    // Clear borders
-    layer.sublayers?.filter  { $0.name == "borderSideLayer" || $0.name == "borderAllSides" }
+    func clearBorders() {
+      layer.borderColor = nil
+      layer.borderWidth = 0
+      layer.sublayers?.filter  { $0.name == "borderSideLayer" || $0.name == "borderAllSides" }
         .forEach { $0.removeFromSuperlayer() }      
-
-    guard let unwrappedBorderColor = borderColor where borderWidth > 0 else {
+    }
+    
+    guard let unwrappedBorderColor = borderColor else {
+      clearBorders()
       return
     }
     
-    if let unwrappedBorderSide = borderSide, side = BorderSide(rawValue: unwrappedBorderSide) {
-      configBorderWithSide(side, borderColor: unwrappedBorderColor)
-    } else {
+    if borderWidth.isNaN || borderWidth <= 0 {
+      clearBorders()
+      return
+    }
+    
+    clearBorders()
```

Comment:
Seems you have merged the changes from upstream master in https://github.com/JakeLin/IBAnimatable/pull/163/commits/b0bbf139c3efb2029f39975969230a8f2d170432 but don't know why our changes didn't apply to your branch. Because we call `clearBorders()` unconditionally. The code in master would be slightly better.

```
    // Clear borders
    layer.sublayers?.filter  { $0.name == "borderSideLayer" || $0.name == "borderAllSides" }
        .forEach { $0.removeFromSuperlayer() }      

    guard let unwrappedBorderColor = borderColor where borderWidth > 0 else {
      return
    }
```
can you rebase again? may need to add the code below back. 

```
layer.borderColor = nil
layer.borderWidth = 0
```